lib/diplomat/rest_client.rb

```diff
@@ -76,6 +76,7 @@ def parse_body
 
     # Get the key/value(s) from the raw output
     def return_value(nil_values=false)
```

Comment:
consul ?keys returns array of keys, so @raw should be ruby array , not sure i understand how it's properly handled by return_value method?